src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/RequestCountyCourtJudgementTest.java

```diff
@@ -51,7 +51,7 @@ public void shouldBeAbleToSuccessfullyRequestCCJ() {
 
         assertThat(updatedCase.getCountyCourtJudgment()).isEqualTo(ccj);
         assertThat(updatedCase.getCountyCourtJudgmentRequestedAt())
-            .isCloseTo(LocalDateTimeFactory.nowInLocalZone(), within(2, ChronoUnit.MINUTES));
+            .isCloseTo(LocalDateTimeFactory.nowInUTC(), within(2, ChronoUnit.MINUTES));
```

Comment:
pretty sure this is wrong, we return the local zone back from the api

Reply:
or some reason we stop timezone information but keep it in the db....
"simpler"

All PDFs receive local british time
The API returns british time

Reply:
changed back